CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Support dynamic title on Windows (Conpty backend only)
```

Comment:
Super tiny bikeshed, but I feel like `Dynamic title support on Windows` would sound a bit better when talking about an 'added feature'. Though I don't *really* care either way, just as a sidenote.

Reply:
No problem, reworded. I kept the note in brackets saying conpty-only. Wasn't sure if you wanted that removed too.